internal/usergroup/add_linux.go

```diff
@@ -164,8 +164,8 @@ func findNextRangeStart(rangeList []user.SubID) (int, error) {
 }
 
 func wouldOverlap(arange user.SubID, ID int64) bool {
-	var low int64 = ID
-	var high int64 = ID + defaultRangeLen
+	var low = ID
+	var high = ID + defaultRangeLen
```

Comment:
While we're updating; to prevent `gofumpt` from changing it, perhaps;

```go
low := ID
high := ID + defaultRangeLen
```

Or (if it's more clear that `high` is low + the default range length)

```go
low := ID
high := low + defaultRangeLen
```